store/mockstore/unistore/cophandler/cop_handler.go

```diff
@@ -60,7 +60,12 @@ func HandleCopRequest(dbReader *dbreader.DBReader, lockStore *lockstore.MemStore
 func HandleCopRequestWithMPPCtx(dbReader *dbreader.DBReader, lockStore *lockstore.MemStore, req *coprocessor.Request, mppCtx *MPPCtx) *coprocessor.Response {
 	switch req.Tp {
 	case kv.ReqTypeDAG:
-		return handleCopDAGRequest(dbReader, lockStore, req, mppCtx)
+		if mppCtx != nil {
+			if mppCtx.TaskHandler != nil {
```

Comment:
How about `if mppCtx != nil && mppCtx.TaskHandler != nil`?

Reply:
make sense and fix it